betterer-clean-reporter.ts

```diff
@@ -0,0 +1,64 @@
+import { BettererReporter } from "@betterer/betterer"
+import chalk from "chalk"
+
+const createCleanReporter = (): BettererReporter => ({
+  runEnd: (runSummary) => {
+    if (runSummary.delta === null) return
+
+    if (runSummary.delta.diff === 0) {
+      console.log(chalk.gray("Nothing to see here."))
+      return
+    }
+    if (runSummary.delta.diff < 0) {
+      console.log(chalk.green("Unwanted things have been removed. Nice!"))
```

Comment:
Maybe something like
```suggestion
      console.log(chalk.green("Cleaned up:"))
```

Reply:
im not spending more time on this reporter 😅. its either `Cleaned up something` and that's it, or `Nice!`.